tests/ami/test_basic.py

```diff
@@ -407,6 +407,78 @@ async def test_read_notification(
     assert response.json()["unread"] is True
 
 
+async def test_stream_notification_events_user_does_not_exist(
+    test_client: TestClient[Litestar],
+) -> None:
+    with test_client.websocket_connect(f"/api/v1/users/{uuid.uuid4()}/notification/events/stream"):
+        # nothing, socket is closed
```

Comment:
Nit: wording: "No failure when the user doesn't exist, no websocket opened".

Also, maybe if you did the code change I proposed (not opening unless we have a confirmed user), maybe make sure that `socket.accept` was never called?
In any case, maybe just make sure that the socket is closed at the end of the test?